labs/lab5/jump-to-here-playbook.yml

```diff
@@ -0,0 +1,38 @@
+--- 
+- hosts: localhost
+  become_method: sudo
+  tasks:
+  - name: build registry
+    command: podman build -t registry /home/ec2-user/summit-2018-container-lab/labs/lab1/registry 
```

Comment:
Once https://github.com/dustymabe/summit-2018-container-lab/pull/36 is merged... the registry directory will live under lab3.

Reply:
#36 has been merged.  We good here?

Reply:
still need to change to this to lab3:
```yaml
command: podman build -t registry /home/ec2-user/summit-2018-container-lab/labs/lab3/registry
```